backend/src/main/java/com/bakdata/conquery/models/query/ManagedQuery.java

```diff
@@ -335,4 +336,9 @@ private static String makeLabelWithRootAndChild(DatasetRegistry datasetRegistry,
 		// Concat everything with dashes
 		return (conceptLabel + "-" + cqConceptLabel).replace(" ", "-");
 	}
+
+	@Override
+	public void visit(Consumer<Visitable> visitor) {
+		query.visit(visitor);
+	}
```

Comment:
Kein `visitor.visit(this)`? Hat gerade keinen use-case, aber eigentlich ist es so gedacht.

Reply:
ups